cypress/integration/task-3.js

```diff
@@ -0,0 +1,46 @@
+import TimeLoggingPage from '../pom/timeLoggingPageObject'
+import NavBar from '../pom/navBarObject'
+
+const timeLoggingPage = new TimeLoggingPage()
+const navBar = new NavBar()
+
+describe('Task 2', function() {
+
+    it('T-2: Users should be able to log in with "Team Lead" role and see appropriate tabs', function () {
+
+        let roles = ["User", "Team Lead", "Manager", "Accountant", "Admin"]
+
+        const nav_length = {
+            "User" : 1,
+            "Team Lead" : 2,
+            "Manager" : 5,
+            "Accountant" : 5,
+            "Admin" : 6,
+        }
```

Comment:
You could also use another approach for role mapping: https://www.javascripttutorial.net/javascript-multidimensional-array/

Reply:
Thanks Nikolaj, I just used the quickest one that was in my mind and I was able to use. Didn't have much time for doing it properly and I agree that it is not the best solution! :D